### PR TITLE
tablet and desktop responsiveness done  

### PR DESCRIPTION
**Pull request by screpyy96**

### CODE REVIEW

File: src/_component/menu/menu.pug [hunk 1]
```diff
@@ -5,7 +5,7 @@
     ul
       each item in data.burger.item 
         li
-          a(href=item.link)= item.title
+          a(style="padding-right:20px")(href=item.link).links= item.title
```
Comment (tandaraelena) [inline src/_component/menu/menu.pug]: Is it still working if you are using the following syntax for this line: 
```a(style="padding-right: 20px" href=`${item.link}`).links= item.title``` ?

**Review by biggicode — APPROVED** [review]
Nice 💯

**Review by tandaraelena — CHANGES_REQUESTED** [review]

File: src/_component/menu/menu.pug (hunk 1 above)
Comment (screpyy96) [reply src/_component/menu/menu.pug]: yes, still working. Thank you

**Review by tandaraelena — APPROVED** [review]

**Review by marianzburlea — COMMENTED** [review]
💯

**Pull request merged**